packages/ui/src/lib/components/Textarea.svelte

```diff
@@ -53,7 +53,8 @@
 		onchange,
 		onfocus,
 		onblur,
-		onkeydown
+		onkeydown,
+		...restProps
```

Comment:
We currently don't use restProps really anywhere in our code base, could we replace this with `oncompositionstart`?